### DISCUSSION TITLE
Send Email Item wise or Connect flow

### DISCUSSION THREAD

**Discussion by sanchitshukla** [microsoft-search/pnp-modern-search#2447] [Q&A]
Can we Send Mail to the Created or any other people picker field value by creating a button or a link for each item.? or if we connect it to a power automate flow how do we get value for a particular row item.?

**Comment by wobba** [comment]
You have to develop a custom web component and use that in your template for customized actions. Unless you have a `<a href="mailto:....">` tag.
